cmd/cdnmirror/main.go

```diff
@@ -0,0 +1,93 @@
+package main
```

Comment:
@der-eismann This is an example of having to programs (`buildutil` and `cdnmirror`) in the same go module.